saucal/woocommerce-gateway-amazon-pay
# Update makepot process to include build js strings

### All Submissions:


### Changes proposed in this Pull Request:

Improve the flows to generate the `.pot` file to include the translatable strings used on Checkout Blocks.

### How to test the changes in this Pull Request:
1. Install a build of this branch.
2. The strings on Checkout Blocks should be translated (e.g. "Proceed to Amazon")

### Changelog entry

> Fix - Translate strings used on Checkout Blocks.
